### ISSUE TITLE
Handler for other file types to upload documents

### ISSUE THREAD

**Issue by ahmdthr** [geosolutions-it/geonode-importer#150]
It would be great if importer can be used to upload documents to GeoNode.

**Comment by giohappy** [comment]
@ahmdthr this is what we expect to happen once this geonode-importer will be officially published and configured inside GeoNode. We don't have plan to adopt it for documents yet, but this is what I wish will happen soon.

**Comment by ahmdthr** [comment]
@giohappy I will contribute for the documents. Any resources or pointers will be greatly appreciated. Is there any mailing-list or chat channels for geonode-importer for developers? Thank you.

**Comment by giohappy** [comment]
>@giohappy I will contribute for the documents. Any resources or pointers will be greatly appreciated. Is there any mailing-list or chat channels for geonode-importer for developers? Thank you.

The best will be through issues once this repo will be moved under GeoNode

**Comment by ahmdthr** [comment]
Is there any timeline for the move to GeoNode? I will try to implement the document upload irrespective on geonode-importer. Wish me luck.

**Comment by giohappy** [comment]
@ahmdthr Could you explain what's your goal? If it's having an API for the upload of documents, it's already available. I thought you meant to replace with an implementation on top of `geonode-importer`

**Comment by ahmdthr** [comment]
Yeah, the goal is to have a common API for all async uploads. I know the document upload endpoint was recently added, but I want to have this with geonode-importer

**Comment by ahmdthr** [comment]
@giohappy I simply wanted to refer to your earlier comment on geonode's document creation PR (https://github.com/GeoNode/geonode/issues/10270#issuecomment-1314191880) We are planning ahead and considering geonode-importer as an official module for geonode. It would be great if we can share resources or any hints for the document upload, I will attempt extending it for the document upload now. Thanks.

**Comment by mattiagiupponi** [comment]
this is going to be implemented in master soon https://github.com/GeoNode/geonode/issues/14503
